tests/test_wave_01.py

```diff
@@ -140,7 +135,6 @@ def test_adds_movie_to_non_empty_user_watchlist():
     assert movie in updated_data["watchlist"]
     assert FANTASY_2 in updated_data["watchlist"]
 
-@pytest.mark.skip()
```

Comment:
[opinion] I appreciate that you removed these instead of just commenting them out.